webapp/packages/core-ui/src/Tabs/TabPanelList.tsx

```diff
@@ -7,24 +7,19 @@
  */
 import { observer } from 'mobx-react-lite';
 import { useContext } from 'react';
-import styled from 'reshadow';
 
-import { useStyles } from '@cloudbeaver/core-blocks';
-import type { ComponentStyle } from '@cloudbeaver/core-theming';
+import { SContext, type StyleRegistry } from '@cloudbeaver/core-blocks';
 
+import { baseTabStyles, tabPanelStyles } from '..';
 import { generateTabElement } from './generateTabElement';
-import { BASE_TAB_STYLES } from './Tab/BASE_TAB_STYLES';
 import { TabPanel } from './TabPanel';
 import type { ITabInfo } from './TabsContainer/ITabsContainer';
 import { TabsContext } from './TabsContext';
 
-interface Props extends React.PropsWithChildren {
-  style?: ComponentStyle;
-}
+const tabPanelRegistry: StyleRegistry = [[tabPanelStyles, { mode: 'append', styles: [baseTabStyles] }]];
 
-export const TabPanelList = observer<Props>(function TabPanelList({ style, children }) {
+export const TabPanelList = observer<React.PropsWithChildren>(function TabPanelList({ children }) {
```

Comment:
this component should not have or use any styles

Reply:
done